src/libraries/Common/src/System/LocalAppContextSwitches.Common.cs

```diff
@@ -56,6 +56,16 @@ private static bool GetSwitchDefaultValue(string switchName)
                 return true;
             }
 
+            if (switchName == "Switch.System.Diagnostics.StackTrace.ShowILOffsets")
```

Comment:
This shared file is meant to be only used to provide defaults for switches that span multiple libraries. It is not the case of this switch.

`AppContextConfigHelper.GetBooleanConfig` may be better way to do this.

Reply:
I will modify to use AppContextConfigHelper.GetBooleanConfig() in LocalAppContextSwitches.cs file.